Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -5,6 +5,8 @@ import MapboxDirections
 // MARK: RouteController Rerouting logic
 /**
  Maximum number of meters the user can travel away from step before `RouteControllerShouldReroute` is emitted.
+ 
+ - important: Only works for `LegacyRouteController`.
```

Comment:
The changelog should mention that this constant no longer has any effect in conjunction with RouteController.